changes/release.go

```diff
@@ -0,0 +1,18 @@
+package changes
+
+type VersionBump struct {
+	Module string
+	From   string
+	To     string
+}
+
+type Release struct {
+	Id            string
```

Comment:
Minor, but for the SDK code base we would typically format this as `ID` rather then `Id`